src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(operation, number = 0) {
+      this.result = operation(this.result, number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(a, b) {
+      return a + b;
+    },
```

Comment:
you can update your operations to work with current value and `this`. Apply for operations below as well
```suggestion
    add(a) {
      this.result += a;
      return this;
    },
```